PyMca5/PyMcaGui/pymca/McaWindow.py

```diff
@@ -74,15 +74,16 @@
 # _logger.setLevel(logging.DEBUG
 
 
-class McaWindow(ScanWindow):
-    def __init__(self, parent=None, name="Mca Window", specfit=None, backend=None,
+class McaWindow(ScanWindow.ScanWindow):
```

Comment:
In coordination with the change above to the `import` statement...

```suggestion
class McaWindow(ScanWindow):
```